docs/create-packages/set-package-type.md

```diff
@@ -9,28 +9,55 @@ ms.topic: conceptual
 
 # Set a NuGet package type
 
-With NuGet 3.5+, packages can be marked with a specific *package type* to indicate its intended use. Packages not marked with a type, including all packages created with earlier versions of NuGet, default to the `Dependency` type.
+Packages can be marked with one more more *package types* to indicate its intended use. Packages not marked with a type, including all packages created with earlier versions of NuGet, default to the `Dependency` type.
 
 - `Dependency` type packages add build- or run-time assets to libraries and applications, and can be installed in any project type (assuming they are compatible).
 
-- `DotnetTool` type packages are extensions to the [dotnet CLI](/dotnet/articles/core/tools/index) and are invoked from the command line. Such packages can be installed only in .NET Core projects and have no effect on restore operations. More details about these per-project extensions are available in the  [.NET Core extensibility](/dotnet/articles/core/tools/extensibility#per-project-based-extensibility) documentation.
+- `DotnetTool` type packages are .NET tools that can be installed by the [dotnet CLI](/dotnet/articles/core/tools/index).
 
 - `Template` type packages provide [custom templates](/dotnet/core/tools/custom-templates) that can be used to create files or projects like an app, service, tool, or class library.
 
 - Custom type packages use an arbitrary type identifier that conforms to the same format rules as package IDs. Any type other than `Dependency` and `DotnetTool`, however, are not recognized by the NuGet Package Manager in Visual Studio.
 
+> [!NOTE]
+> Support for package types was added in NuGet 3.5.
+> If you don't need a custom package type, it's best to *not* explicitly set the package type. NuGet defaults to `Dependency` this type when no type is specified.
+
+# [Using dotnet CLI](#tab/dotnet)
+
+Package types can be set in the project file (`.csproj`):
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
+    
+    <PackageType>MyCustomType, 1.0.0.0;Dependency, 1.0.0.0</PackageType>
```

Comment:
Open questions:
* What is the right package type version for Dependency? The example [here](https://review.docs.microsoft.com/en-us/nuget/reference/msbuild-targets?branch=pr-en-us-2397#pack-target) ues version `2.0.0.0` for Dependency - is that correct?
* Should package type versions have segments? I followed the existing reference docs

Reply:
Dependency version is `0.0` (default).

What do you mean by segments?

This example is dumb. There is no such thing as Dependency 1.0.0.0 or 2.0.0.0 today.

Reply:
I think a better example is just `MyCustomType, 1.0.0.0;Dependency`.

Reply:
4 segments as in `1.0.0.0` vs 3 as in `1.0.0`. Should it be 2 segments???

Reply:
Updated to `MyCustomType, 1.0.0.0;Dependency`

Reply:
Don't get me started on System.Version vs. NuGetVersion
https://sharplab.io/#v2:C4LgTgrgdgNAJiA1AHwAICYAMBYAUBgRjz1QIE4AKKAUwHcACANWrAGcBLAeygoCJMAdJl4BKegF5x9Gg2ZsuPfkKGiRAbiA

Reply:
In other words, people can choose whatever segments they want and it has different meaning, at least w.r.t. to comparison.

Reply:
![image](https://user-images.githubusercontent.com/737941/115499659-a541b000-a224-11eb-9c24-c29291c8b0b3.png)